mlflow/utils/file_utils.py

```diff
@@ -967,3 +969,26 @@ def chdir(path: str) -> None:
         yield
     finally:
         os.chdir(cwd)
+
+
+def get_total_file_size(path: str) -> int:
+    """
+    Return the size of all files under given path, including files in subdirectories.
+
+    :param path: The absolute path of a local directory.
+    :return: size in bytes.
+    """
+    if not os.path.exists(path):
+        raise MlflowException(
+            message=f"The given {path} does not exist.", error_code=INVALID_PARAMETER_VALUE
+        )
+    if not os.path.isdir(path):
+        raise MlflowException(
+            message=f"The given {path} is not a directory.", error_code=INVALID_PARAMETER_VALUE
+        )
```

Comment:
Do we need these checks? Doesn't `os.walk` throw?

Reply:
No, i remember os.walk just return empty iterator when given path is not a dir.